lnwallet/channel.go

```diff
@@ -3607,6 +3638,14 @@ func (lc *LightningChannel) createCommitDiff(
 				PaymentHash: pd.RHash,
 			}
 			copy(htlc.OnionBlob[:], pd.OnionBlob)
+			if pd.BlindingPoint != nil {
```

Comment:
Similar point here re just copying over the raw bytes so you don't need to be concerned about the record mapping at this state.